vm/Cargo.toml

```diff
@@ -12,3 +12,4 @@ nohelmet = []
 
 [dependencies]
 unicode-segmentation = "1.10.1"
+bridge_types = { workspace = true }
```

Comment:
This is cool but we will need to stay vigilant that bridge_types remains a lean no-dep crate.

Reply:
I promise!